Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int[] arr = new int[]{1, 2, 3, 4, 5};
+        System.out.println("Even numbers:");
+
+        for (int i = 0; i < arr.length; i = i * 2)

```

Comment:
**Security Issue**: The loop increment 'i = i * 2' in the method Even() can potentially lead to an infinite loop if 'i' is initialized to 0, as multiplying by 2 will always result in 0, causing the loop to never terminate. This can lead to a Denial of Service (DoS) if this code is executed in a server environment, where an infinite loop could consume excessive CPU resources. <br> **Fix**: Change the loop increment to ensure it will eventually terminate. If the intention is to print even-indexed elements, the increment should be 'i = i + 2' instead of 'i = i * 2'. <br> **Code Suggestion**: 
 ```
 for (int i = 0; i < arr.length; i += 2)
 {
     System.out.println(arr[i]);
 }
 ```